cmp/filter/FilterChooserModel.js

```diff
@@ -517,4 +554,5 @@ export class FilterChooserModel {
  * @typedef {Object} FilterChooserPersistOptions
  * @extends PersistOptions
  * @property {boolean} [persistValue] - true (default) to save value to state.
+ * @property {boolean} [persistFavorites] - true (default) to include favorites.
```

Comment:
Would you ever set persistFavorites to false, and have the persisted elsewhere?  Perhaps.  Trying to keep this as much like DimChooser History as possible -- but the external control for favorites definitely compexifies things.